src/views/dashboard/Dashboard.js

```diff
@@ -177,133 +178,134 @@ const Dashboard = () => {
 
   return (
     <>
-      <CCard className="mb-4">
```

Comment:
Can we move this out to a template file for easy refactoring later?

Reply:
This is actually just place holder, it should be deleted.

Reply:
@lakivisi The actual dashboard will come at a later point. In this change we are mainly introducing the `<ComponentRBAC />` component and logic.

Reply:
Yes, the component will nest other components that we need to control access to.